# HG changeset patch
# User Rik <rik@octave.org>
# Date 1428634401 25200
#      Thu Apr 09 19:53:21 2015 -0700
# Node ID ab7582167810354cfc56506a6a03f176cc21dee8
# Parent  654705239575d40483ef696bc51f03f6396cc3c7
Fix failing BIST test for tilde_expand() on Windows (bug #44796).

* sysdep.cc (Ftilde_expand): Don't use fullfile in BIST test as it
replaces all slashes in path with the current file separator.

diff --git a/libinterp/corefcn/sysdep.cc b/libinterp/corefcn/sysdep.cc
--- a/libinterp/corefcn/sysdep.cc
+++ b/libinterp/corefcn/sysdep.cc
@@ -999,17 +999,17 @@ tilde_expand (\"~/bin\")\n\
     print_usage ();
 
   return retval;
 }
 
 /*
 %!test
 %! home = get_home_directory ();
-%! assert (tilde_expand ("~/foobar"), fullfile (home, "/foobar"));
+%! assert (tilde_expand ("~/foobar"), [home "/foobar"]);
 %! assert (tilde_expand ("/foo/bar"), "/foo/bar");
 %! assert (tilde_expand ("foo/bar"), "foo/bar");
 */
 
 DEFUN (get_home_directory, , ,
        "-*- texinfo -*-\n\
 @deftypefn {Built-in Function} {@var{homedir} =} get_home_directory ()\n\
 Return the current home directory.\n\
